src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -182,26 +188,27 @@ def safe_for_comment(
                 OPEN_PR_METRICS_BASE.format(key="rejected_comment"),
                 tags={"reason": "too_many_files"},
             )
-            return False, []
+            return []
         if changed_lines_count > OPEN_PR_MAX_LINES_CHANGED:
             metrics.incr(
                 OPEN_PR_METRICS_BASE.format(key="rejected_comment"),
                 tags={"reason": "too_many_lines"},
             )
-            return False, []
+            return []
 
-    return safe_to_comment, filtered_pr_files
+    return filtered_pr_files
 
 
-def get_pr_filenames_and_patches(pr_files: List[Dict[str, str]]) -> Tuple[List[str], List[str]]:
+def get_pr_filenames_and_patches(pr_files: List[Dict[str, str]]) -> List[PullRequestFile]:
```

Comment:
Is this still returning `filenames_and_patches`? or is it just retunring `files`?

Reply:
PullRequestFile contains filenames and patches

Reply:
i renamed the function